src/fillTank.test.js

```diff
@@ -1,11 +1,98 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+   const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared ', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should calculate full tank If the amount is not given', () => {
+    const customer = {
+      money: 3000, // customer account balance
+      vehicle: {
+        maxTankCapacity: 40, // fuel tank volume
+        fuelRemains: 0, // Remaining fuel in the tank
+      },
+    };
+
+    fillTank(customer, 30);
```

Comment:
Here, the fillTank function is called with an amount of 30, which contradicts the test description that states the amount is not given. If you want to test the behavior when no amount is provided, you should call fillTank without the amount parameter.

Reply:
30 is fuel price not amount
Function fillTank(customer, fuelPrice, amount = Infinity)